yarn.lock

```diff
@@ -1430,10 +1430,10 @@
   dependencies:
     "@sinonjs/commons" "^3.0.0"
 
-"@stripe/connect-js@2.0.1-beta.7":
-  version "2.0.1-beta.7"
-  resolved "https://registry.yarnpkg.com/@stripe/connect-js/-/connect-js-2.0.1-beta.7.tgz#efa2d73c66d8ffb306b057f84f71f9c84488cc5d"
-  integrity sha512-c0MED84WJydlTwILsYBeQ1y5ApwNQpoIELZbLInEj2UN2BwPr14MAwoGcmtAIsYRWCfgy0RkLljzn2lqaLkVTA==
+"@stripe/connect-js@3.1.0-beta.1":
```

Comment:
PR looks good, however is mising the package.json version bumps (in both the devDependencies and peerDependencies). We should also wait for https://github.com/stripe/react-connect-js/pull/53 to merge first